contrib/src/sceneeditor/collisionWindow.py

```diff
@@ -294,7 +294,7 @@ def createEntryField(self, parent, catagory, id, value,
         widget.configure(entry_state = initialState)
         widget.pack(side=LEFT)
         self.widgetDict[catagory+id] = widget
-        if defaultButton and (defaultFunction!=None):
+        if defaultButton and (defaultFunctionis notNone):
```

Comment:
@mgracer48 ```!= None``` and ```is not None``` are the same exact thing, what's the point in changing it to begin with?